PostHogExample/AppDelegate.swift

```diff
@@ -12,18 +12,20 @@ import UIKit
 class AppDelegate: NSObject, UIApplicationDelegate {
     func application(_: UIApplication, didFinishLaunchingWithOptions _: [UIApplication.LaunchOptionsKey: Any]? = nil) -> Bool {
         let config = PostHogConfig(
-            apiKey: "phc_WKfvDfedaJEDCoUmt9pVa3OWtbbUP1W2ctxwXkt3A3n"
+            apiKey: "phc_DOkauJvMj0YFtJsPHqzH6BgpFm79CvU9DPE5E22yRMk",
+            host: "http://localhost:8010"
         )
         // the ScreenViews for SwiftUI does not work, the names are not useful
         config.captureScreenViews = false
         config.captureApplicationLifecycleEvents = false
 //        config.flushAt = 1
 //        config.flushIntervalSeconds = 30
         config.debug = true
+        config.flushAt = 1
         config.sendFeatureFlagEvent = false
 
         #if os(iOS)
-            config.sessionReplay = true
+            config.sessionReplay = false
```

Comment:
same as https://github.com/PostHog/posthog-ios/pull/404/files#r2630728120
all features enabled by default - its easier to catch bugs